network-api/networkapi/templates/partials/footer.html

```diff
@@ -45,19 +45,28 @@ <h5 class="h5-heading">More about us</h5>
           <div class="col-lg-6 col-xl-7">
             <ul class="link-list list-unstyled mb-0">
               {% block general_links %}
-              <li class="mb-2"><a id="donate-footer-btn" href="https://donate.mozilla.org/?utm_source=foundation.mozilla.org&utm_medium=referral&utm_content=footer" target="_blank" rel="noopener noreferrer" class="dark-theme">Donate</a></li>
-              <li class="mb-2"><a href="https://www.mozilla.org/about/legal/terms/mozilla/" class="dark-theme">Legal</a></li>
-              <li class="mb-2"><a href="https://creativecommons.org/licenses/by/4.0" class="dark-theme">License</a></li>
-              <li class="mb-2"><a href="https://www.mozilla.org/about/governance/policies/participation/" class="dark-theme">Participation Guidelines</a></li>
-              <li class="mb-2"><a href="https://mozilla.org/privacy/websites/" class="dark-theme">Privacy</a></li>
-              <li class="mb-2"><a href="https://www.mozilla.org/privacy/websites/#cookies" class="dark-theme">Cookies</a></li>
+              <li><a id="donate-footer-btn" href="https://donate.mozilla.org/?utm_source=foundation.mozilla.org&utm_medium=referral&utm_content=footer" target="_blank" rel="noopener noreferrer" class="dark-theme">Donate</a></li>
+              <li><a href="https://www.mozilla.org/about/legal/terms/mozilla/" class="dark-theme">Legal</a></li>
+              <li><a href="https://creativecommons.org/licenses/by/4.0" class="dark-theme">License</a></li>
+              <li><a href="https://www.mozilla.org/about/governance/policies/participation/" class="dark-theme">Participation Guidelines</a></li>
+              <li><a href="https://mozilla.org/privacy/websites/" class="dark-theme">Privacy</a></li>
+              <li><a href="https://www.mozilla.org/privacy/websites/#cookies" class="dark-theme">Cookies</a></li>
               {% endblock %}
             </ul>
           </div>
         </div>
+        {% if exclude_language_switcher != True %}
```

Comment:
We should set this to False in the buyersguide `bg_base.html`, so that it does not include the switched (since PNI is english-only and we rigorously redirect for it no matter which locale you try to use)